src/locale/locales/fi/messages.po

```diff
@@ -345,11 +345,11 @@ msgstr "Sovellussalasanat"
 #: src/components/moderation/LabelsOnMeDialog.tsx:134
 #: src/components/moderation/LabelsOnMeDialog.tsx:137
 msgid "Appeal"
-msgstr ""
+msgstr "Valita"
 
 #: src/components/moderation/LabelsOnMeDialog.tsx:202
 msgid "Appeal \"{0}\" label"
-msgstr ""
+msgstr "Valita \"{0}\" label"
```

Comment:
muualla oli käytetty labelista suomennosta "merkintä", tässäkin ehkä voisi?

Reply:
Tää on näitä kohtia, joissa ei näe kontekstia. Ehdotatko siis käännökseksi `Valita \"{0}\" merkintä`?

Reply:
Eikun joo, nehän liittyy tähän: [https://bsky.app/profile/moderation.bsky.app](https://bsky.app/profile/moderation.bsky.app) . Mikähän tossa ois fiksuin käännös. Oletan, että kyseessä on moderointipäätöksestä tms valittaminen. "Valita merkinnästä"? @mmyllynen

Reply:
Tuolla on näemmä tota osiota kokonaan kääntämättä. 

`#: src/components/moderation/LabelsOnMe.tsx:59
msgid "label has been placed on this {labelTarget}"
msgstr ""

#: src/components/moderation/ContentHider.tsx:144
msgid "Labeled by {0}."
msgstr ""

#: src/components/moderation/ContentHider.tsx:142
msgid "Labeled by the author."
msgstr ""

#: src/view/screens/Profile.tsx:186
msgid "Labels"
msgstr ""

#: src/screens/Profile/Sections/Labels.tsx:143
msgid "Labels are annotations on users and content. They can be used to hide, warn, and categorize the network."
msgstr ""

#: src/components/moderation/LabelsOnMe.tsx:61
msgid "labels have been placed on this {labelTarget}"
msgstr ""

#: src/components/moderation/LabelsOnMeDialog.tsx:63
msgid "Labels on your account"
msgstr ""

#: src/components/moderation/LabelsOnMeDialog.tsx:65
msgid "Labels on your content"
msgstr ""

#: src/components/moderation/ContentHider.tsx:65
#: src/components/moderation/ContentHider.tsx:128
msgid "Learn more about the moderation applied to this content."
msgstr ""
`

Reply:
Joo, tämä kuulostaisi ehkä järkevimmältä tällä tiedolla. Pitäisi kaivaa koodista miten tuota on käytetty jos haluisi varmemman tiedon, mutta ei nyt ole kyllä sellaiseen aikaa tässä kohtaa. Sanoisin että parannus tuo nyt ainakin olisi kuin puhtaasti kääntämätön "label".

Reply:
Fixed.